platform/windows/display_server_windows.cpp

```diff
@@ -1513,6 +1522,187 @@ Ref<Image> DisplayServerWindows::screen_get_image_rect(const Rect2i &p_rect) con
 	return img;
 }
 
+#ifdef D3D12_ENABLED
+static bool _get_screen_desc(int p_screen, DXGI_OUTPUT_DESC1 &r_Desc) {
+	ComPtr<IDXGIFactory4> dxgi_factory;
+	r_Desc = {};
+
+	if (FAILED(CreateDXGIFactory2(0, IID_PPV_ARGS(&dxgi_factory)))) {
+		return false;
+	}
+
+	// Retrieve the current default adapter.
+	ComPtr<IDXGIAdapter1> dxgiAdapter;
+	if (FAILED(dxgi_factory->EnumAdapters1(0, &dxgiAdapter))) {
+		return false;
+	}
+
+	UINT i = 0;
+	ComPtr<IDXGIOutput> screenOutput;
+	while (dxgiAdapter->EnumOutputs(i, &screenOutput) != DXGI_ERROR_NOT_FOUND) {
+		if (i == (UINT)p_screen) {
+			break;
+		}
+
+		i++;
+	}
+
+	ComPtr<IDXGIOutput6> output6;
+	if (FAILED(screenOutput.As(&output6))) {
```

Comment:
```suggestion
	if (screenOutput == nullptr || FAILED(screenOutput.As(&output6))) {
```

This is the AMD integrated graphics driver crash that I reported [here}(https://github.com/godotengine/godot/pull/94496#issuecomment-2761720109) and later [here as a regression](https://github.com/godotengine/godot/pull/94496#issuecomment-3009080786).

I haven't done a review of this code, so it's possible this is not the correct fix! I was able to at least confirm that it prevents the crash, but a different and better solution might be needed.